requirements.txt

```diff
@@ -1,4 +1,4 @@
 regex
 enchant
-python-gtkspellcheck
+gspell
```

Comment:
Isn't this a system dependency? pip doesn't find this.

Reply:
Ah, I didn't know that was only for pip deps. Just assumed it was a list of general dependencies.